lib/toxiproxy.rb

```diff
@@ -283,6 +291,35 @@ def toxics
 
   private
 
+  def version_string
+    return @version_string if @version_string
+
+    version_response = self.class.version
```

Comment:
Just so i'm understanding this correctly - self.class.version contains a JSON blob that it gets from the server?

Reply:
Yeah! It turns out `Toxiproxy.version` is already exposed. This method will hit the underlying toxiproxy server's `/version` endpoint and return the result as a string. 

https://github.com/Shopify/toxiproxy-ruby/blob/f28165740414e37f53e02ab075a47b1ed3eb9e94/lib/toxiproxy.rb#L49-L56

That return will be a JSON blob, and while I wanted to parse and extract `version` from it all there that'd be a breaking change for anyone who already doing that work. It's probably something could change in a major release because while folks might be doing that, I think they'd prefer if the library did it for them :D